frostsnap_comms/src/lib.rs

```diff
@@ -370,13 +379,29 @@ impl<B: Gist> Gist for DeviceSendMessage<B> {
 #[derive(Encode, Decode, Debug, Clone)]
 pub enum DeviceSendBody {
     Core(frostsnap_core::message::DeviceToCoordinatorMessage),
-    Debug { message: String },
-    Announce { firmware_digest: Sha256Digest },
-    SetName { name: String },
+    Debug {
+        message: String,
+    },
+    Announce {
+        firmware_digest: Sha256Digest,
+    },
+    SetName {
+        name: String,
+    },
     DisconnectDownstream,
     NeedName,
     _LegacyAckUpgradeMode, // Used by earliest devices
     Misc(CommsMisc),
+    Announce2 {
```

Comment:
Is this a temporary or longer-term approach to message versioning?
Will we bundle these all up into a higher enum on the model level?

Reply:
> Is this a temporary or longer-term approach to message versioning?

long term

>  Will we bundle these all up into a higher enum on the model level?

What's the "model level"?

I should probably delete this enum variant since I didn't end up using `Announce2` for much.